scrapi/requests.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
```

Comment:
Could you docstring this file/functions/classes? Probably confusing if you hadn't already told why you did it